src/components/experiment-tracking/run-metadata/run-metadata.js

```diff
@@ -117,6 +117,7 @@ const RunMetadata = ({
                   <tr>
                     <td className="details-metadata__title" colSpan="2">
                       <span
+                        className="details-metadata__title-detail"
```

Comment:
This is to fix the title font not being correct here (second edit): 

<img width="924" alt="image" src="https://user-images.githubusercontent.com/16869061/175047570-93068dc5-9c63-4a78-9b24-50e52c00d888.png">